project-base/storefront/urql/cache/updates.ts

```diff
@@ -77,7 +77,7 @@ export const cacheUpdates: UpdatesConfig = {
             _args: ChangeTransportInCartMutationVariablesApi,
             cache,
         ) {
-            if (result.ChangeTransportInCart?.uuid) {
+            if (result.ChangeTransportInCart?.__typename) {
```

Comment:
Is it necessary to check for the `__typename`? Cannot we have `if (result.ChangeTransportInCart) {`?

Reply:
I don't know why I haven't thought of this before 🤦 thanks for the suggestion